src/components/molecules/Visualizer/Plugin/context.tsx

```diff
@@ -108,6 +110,14 @@ export function Provider({
   const getLayerSelectionReason = useGet(layerSelectionReason);
   const getLayerOverriddenInfobox = useGet(layerOverridenInfobox);
   const getLayerOverriddenProperties = useGet(layerOverriddenProperties);
+  const getBuiltinExtensionIds = useCallback(
+    () =>
+      builtinPrimitives
```

Comment:
Getting a CI linting warning here. Make sure to save locally this file if you accepted changes by @rot1024 directly in github. Sometimes the formatting isn't correct.